package.json

```diff
@@ -1,30 +1,33 @@
 {
   "name": "setup-nomad-pack",
   "version": "0.9.1",
-  "description": "A GitHub Action to install a specific version of HashiCorp Nomad-Pack and add it to PATH.",
-  "main": "lib/index.js",
+  "description": "A GitHub Action to install a specific version of HashiCorp Nomad Pack and add it to PATH.",
+  "main": "src/index.ts",
```

Comment:
typically main is specified as an entry point that is to be executed, so that should be dist

Reply:
Great catch! I'v set this to `dist/index.js`